README.md

```diff
@@ -1,2 +1,53 @@
 # MoSDeF-dihedral-fit
-MoSDeF-dihedral-fit: A simple software package to fit dihedrals via the MoSDeF software
+MoSDeF-dihedral-fit: A simple software package to fit dihedrals via the MoSDeF software.
+
+This package is a tool to add dihedral parameters for [Ryckaert-Bellemans](https://manual.gromacs.org/current/reference-manual/functions/bonded-interactions.html#proper-dihedrals-ryckaert-bellemans-function) dihedral based forcefields by fitting QM calculations for the rotation about the dihedral angle to this functional form. The main functionality is found through the following code snippet.
+```python
```

Comment:
We should remove this, as we always forget to change these.

**Let's just leave the link to the examples below, removing the example here**
https://github.com/GOMC-WSU/GOMC_Examples/tree/main/MoSDeF-dihedral-fit

Reply:
You mean remove the whole code snippet or just the surrounding text? I would suggest at least having some example of the API because that's a huge selling point. No one wants to use something that they can't understand how to get started, so showing them it's a single function call would be nice. Linking to the examples hides that selling point behind a few more clickable links that a casual user may or may not actually get to.

Reply:
I also don't think this API should change. And if it does, the examples have to change too so changing this one function call that's actually part of the main repo would be easy to do in a single PR as opposed to changing code in the example too.

Reply:
My point was all this is in the example in the docs and on the wsu file examples.  We should just put a link.  Otherwise, we have to change it 3 times if it changes (in different locations).  I think a link is ok, as it is 1 click.

Reply:
If you want to add the print file to the WSU examples and the docs, I am ok with that

Reply:
I meant remove the code example and just put a link to the examples